pkg/sqlcmd/sqlcmd.go

```diff
@@ -86,10 +87,13 @@ func New(l Console, workingDirectory string, vars *Variables) *Sqlcmd {
 	s.PrintError = func(msg string, severity uint8) bool {
 		return false
 	}
+	s.SetOutput(os.Stdout)
+	s.SetError(os.Stderr)
```

Comment:
>s.SetError(os.Stderr) [](http://example.com/codeflow?start=1&length=21)

this is a change in behavior for any other places that call GetError(), now it's going to return stderr by default instead of stdout. Please verify that's the right behavior in those other places.

Reply:
This functionality change is leading to undesired side effects such as redirecting server errors to stderr as well. (I will close this PR and create new one)
I have reworked the changes to keep existing functionality as is in PR #143 and redirecting only the sqlcmd errors which is what the ODBC is also doing.